packages/regl-worldview/src/commands/GLTFScene.js

```diff
@@ -280,8 +282,8 @@ export default class GLTFScene extends React.Component<Props, {| loadedModel: ?O
             <Command
               {...rest}
               reglCommand={drawModel}
-              drawProps={sharedDrawProps}
-              hitmapProps={drawHitmap ? { ...sharedDrawProps, drawHitmap } : undefined}
+              drawProps={{ ...restOfChildren, model: loadedModel }}
```

Comment:
Could just use `...children, id: null` and then you don't need `restOfChildren`.